indykite_sdk_node/src/sdk/model/config/email/factory.ts

```diff
@@ -1,17 +1,18 @@
 import * as grpc from '../../../../grpc/indykite/config/v1beta1/model';
 import { SdkErrorCode, SdkError } from '../../../error';
 import { EmailMessage } from './message';
-import { EmailProvider } from './provider';
-import { SendgridEmailProvider } from './providers/sendgrid';
+import { EmailService } from './service';
+import { SendgridEmailService } from './providers/sendgrid';
 import { EmailTemplate } from './template';
 
-export type EmailProviderType = SendgridEmailProvider;
+export type EmailServiceConfigType = SendgridEmailService;
+export const EMAIL_SERVICE_CONFIG = "emailServiceConfig";
 
 type EmailDefinitionType =
   | { oneofKind: 'template'; template: EmailTemplate }
   | { oneofKind: 'message'; message: EmailMessage };
 
-export class EmailProviderFactory {
+export class EmailServiceFactory {
   private static createMessageOrTemplate(def: EmailDefinitionType): EmailMessage | EmailTemplate {
     if (def.oneofKind === 'message') {
       const e = def.message as grpc.EmailMessage;
```

Comment:
Doesn't give any errors when doing this

```suggestion
      const e = def.message;
```